package.json

```diff
@@ -12,13 +12,19 @@
     "electron-prebuilt-compile": "^1.2.5",
     "eslint": "^3.0.1",
     "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.11.0",
+    "eslint-plugin-jsx-a11y": "^2.0.1",
+    "eslint-plugin-react": "^5.2.2",
```

Comment:
Haha I think my Travis config file specifies node 6. Check a build, it spits out node, npm, and nvm version numbers.